ftw/htmlblock/browser/resources/styles.scss

```diff
@@ -1 +1,15 @@
 @include portal-type-font-awesome-icon(ftw-htmlblock-htmlblock, code);
+
+.sl-can-edit .ftw-htmlblock-htmlblock {
```

Comment:
Fixes in ftw.simplelayout. `sl-can-edit` is only available if the user can modify + `disable_editmode` is false